json_merger/config.py

```diff
@@ -85,5 +85,6 @@ class UnifierOps(object):
         'KEEP_HEAD_ENTITIES_CONFLICT_ON_NEW_UPDATE'
     ]
 
+
```

Comment:
redundant

Reply:
I think this is required by `flake8` (two newlines after class definition).